setup.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import absolute_import
-
+from copy import deepcopy
```

Comment:
why do you need this in `setup.py`?

Reply:
i was confused, considering pull request checklist, i will remove it.